### PR TITLE
Use protonDB svg, encoded as base64

### PR DESCRIPTION
**Pull request by vrman123**
source: https://www.protondb.com/sites/protondb/images/site-logo.svg

probably best to ask the protonDB people for permission, but they did already give permission to joam*3

### CODE REVIEW

**Comment by vrman123** [comment]
image
screenshot for clarity

**Comment by OMGDuke** [comment]
Screenshot looks good. Base64 appears valid but I'm getting an error when it tries to load 

(Shortened  svg string for brevity)
```GET data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iM...Zz4KPC9zdmc+Cg==' net::ERR_INVALID_URL```

**Comment by OMGDuke** [comment]
Seems removing the single quote at the end fixes it for me

File: src/components/protonMedal.tsx [hunk 1]
```diff
@@ -73,7 +72,7 @@ export default function ProtonMedal(
...
...zItNTMuNS00Ni40YzAtMTQuNCwyMS0zMi43LDUzLjUtNDYuNGM3LjktMy40LDE2LjUtNi40LDI1LjQtOS4zYzUuMiwxOCwxMi4xLDM2LjcsMjAuNiw1NS45CgkJYy04LjQsMTkuMS0xNS4yLDM3LjctMjAuNCw1NS42QzExMC45LDI5OC41LDEwMi4zLDI5NS40LDk0LjMsMjkyeiBNMTQzLjcsNDIzYy0xMi41LTcuMi0xNy45LTM0LjQtMTMuNy02OS40CgkJYzEtOC42LDIuNy0xNy43LDQuNy0yN2MxOCw0LjQsMzcuNiw3LjgsNTguMiwxMGMxMi40LDE3LDI1LjIsMzIuNCwzOC4yLDQ1LjljLTI5LjksMjcuOC01OCw0My03Nyw0MwoJCUMxNDkuOSw0MjUuNCwxNDYuNCw0MjQuNiwxNDMuNyw0MjN6IE0zNjEuMywzNTMuMWM0LjMsMzUtMSw2Mi4zLTEzLjQsNjkuNWMtMi44LDEuNy02LjMsMi40LTEwLjUsMi40Yy0xOSwwLTQ3LjEtMTUuMS03Ny00Mi43CgkJYzEyLjgtMTMuNSwyNS43LTI4LjgsMzcuOS00NS44YzIwLjctMi4yLDQwLjQtNS42LDU4LjMtMTAuMUMzNTguNiwzMzUuNywzNjAuMiwzNDQuNiwzNjEuMywzNTMuMXogTTM5Ni42LDI5MgoJCWMtNy45LDMuNC0xNi41LDYuNC0yNS40LDkuM2MtNS4yLTE4LTEyLjEtMzYuNy0yMC42LTU1LjljOC40LTE5LjEsMTUuMi0zNy43LDIwLjQtNTUuNmM5LjEsMi44LDE3LjcsNiwyNS44LDkuNAoJCWMzMi41LDEzLjksNTMuNSwzMiw1My41LDQ2LjRDNDUwLDI1OS45LDQyOSwyNzguMiwzOTYuNiwyOTJ6Ii8+Cgk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNMTUzLjYsNDUuNUwxNTMuNiw0NS41TDE1My42LDQ1LjV6Ii8+Cgk8Y2lyY2xlIGNsYXNzPSJzdDIiIGN4PSIyNDUuNCIgY3k9IjI0NS41IiByPSI0MS45Ii8+Cgk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNMzM2LjgsNDUuMkwzMzYuOCw0NS4yTDMzNi44LDQ1LjJ6Ii8+CjwvZz4KPC9zdmc+Cg=='" />
```
Comment (OMGDuke) [inline src/components/protonMedal.tsx:75]: Does the single quote at the end of this line serve a purpose?

`Cg=='"`

it seems to break the image for me

**Comment by OMGDuke** [comment]
Currently using `red` for the Borked background

image

Could add a background for certain medals? Any other ideas?

File: src/components/protonMedal.tsx (hunk 1 above)
Comment (OMGDuke) [inline src/components/protonMedal.tsx:75]: The linting that's setup on the project will show an error for img tags without an alt attrib.

Maybe `ProtonDB` or `ProtonDB Logo`? What do you think?

Comment (vrman123) [reply src/components/protonMedal.tsx:75]: probably 'ProtonDB Logo'

Comment (vrman123) [reply src/components/protonMedal.tsx:75]: not sure how that snuck in there, removing it seems to fix it?

**Comment by vrman123** [comment]
> Currently using `red` for the Borked background
> 
> image
> 
> Could add a background for certain medals? Any other ideas?

not sure about this one.
maybe making the logo fully white for borked?
should be easily possible with the 'filter' tag

**Comment by OMGDuke** [comment]
> \nmaybe making the logo fully white for

That could work. Depends if ProtonDB have strict brand and don't want people recolouring their logo :p but I was already showing it as all white with the react-icon so probably fine

**Comment by vrman123** [comment]
> That could work. Depends if ProtonDB have strict brand and don't want people recolouring their logo :p but I was already showing it as all white with the react-icon so probably fine

image
looks like this (ignore rounded corners), not bad imo

**Comment by joamjoamjoam** [comment]
You can also store it locally and bundle it in assets. I had that working in my repo.

**Pull request merged**